src/core/grid_based_algorithms/lb_interface.hpp

```diff
@@ -86,25 +86,29 @@ void lb_lbfluid_sanity_checks();
 /**
  * @brief Set the LB density for a single node.
  */
-void lb_lbnode_set_density(const Utils::Vector3i &ind, double density);
+void lb_lbnode_set_density(const Utils::Vector3i &ind, double density,
```

Comment:
you are definitely mixing up different levels of abstraction here

Reply:
true, I came up with a more elegant solution in  069e38ba14d6